corehq/apps/app_manager/util.py

```diff
@@ -119,37 +119,34 @@ def split_path(path):
 
 
 def save_xform(app, form, xml):
```

Comment:
Oh you know what, I wonder if this wasn't being called on a `ShadowForm`, but that `change_xmlns()` below _was_ being called on a shadow form, because as you pointed out it seems to try to modify other forms in the app.